src/components/a-choice/index.js

```diff
@@ -0,0 +1,55 @@
+import classnames from 'classnames';
+import { BaseComponentGlobal } from '../_abstract/component-types';
+// import the styles used for this component
+import styles from './index.scss';
+// import the template used for this component
+import template from './_template';
+import wcdomready from '../../js/wcdomready';
+import getAttribute from '../../js/get-attribute';
+
+class AXAChoice extends BaseComponentGlobal {
+  constructor() {
+    super(styles, template);
+
+    // does this provide context (See docs for context) ?
+    // this.enableContext()
+
+    // or do you want to consume a specific context
+    // this.selectContext('axa-context-provider');
+  }
+
+  /**
+   * REF: https://www.w3.org/TR/custom-elements/#custom-element-conformance
+   */
+  connectedCallback() {
+    super.connectedCallback();
+
+
+    const choiceClasses = classnames(this.initialClassName, 'a-choice', {
+      'a-choice--error': getAttribute(this, 'error'),
```

Comment:
@TheDadi 
If you need multiple attributes you can replace this by a single call to `getAttributes`.
**Check-out** may missing camelcase fix #277